iphone/Maps/Categories/CALayer+SetCorner.swift

```diff
@@ -1,13 +1,9 @@
 extension CALayer {
   func setCorner(radius: CGFloat,
-                 curve: CALayerCornerCurve? = nil,
                  corners: CACornerMask? = nil) {
     cornerRadius = radius
     if let corners {
       maskedCorners = corners
     }
-    if #available(iOS 13.0, *) {
-      cornerCurve = curve ?? .continuous
-    }
```

Comment:
Why this code was removed?
Is the continuous corner curve is default in the latest version?

(I haven't updated yet to the Xcode 16)

Reply:
XCode complained that a parameter `curve: CALayerCornerCurve? = nil,` is only available since iOS 13.

I didn't find any place in the code that uses this parameter. Am I missing something?

Reply:
This method was designed such way that continuous curve will be used for the all corners if the other option isn't provided. 

This yodate will breaks up all continuous curves and returns to the circulars (default) that are not so smooth and looks not so good. 

Please see #7380

Reply:
@kirylkaveryn fixed, PTAL